### ISSUE TITLE
Murks

### ISSUE THREAD

**Issue by Thomapressbaum** [mohammadnakhaee/tbstudio#6]
Programm starts opening but then I get the following error message:

(tbstudio:7): dbind-WARNING **: 14:25:48.698: Couldn't connect to accessibility bus: Failed to connect to socket /run/user/1000/at-spi/bus_1: No such file or directory
libGL error: No matching fbConfigs or visuals found
libGL error: failed to load driver: swrast

(tbstudio:7): Gtk-WARNING **: 14:26:02.169: Negative content height -1 (allocation 1, extents 1x1) while allocating gadget (node notebook, owner GtkNotebook)

(tbstudio:7): Gtk-CRITICAL **: 14:26:02.169: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkNotebook

(tbstudio:7): Gtk-CRITICAL **: 14:26:02.183: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkScrollbar

(tbstudio:7): Gtk-CRITICAL **: 14:26:02.183: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkScrollbar

(tbstudio:7): Gtk-WARNING **: 14:26:02.267: Negative content height -1 (allocation 1, extents 1x1) while allocating gadget (node notebook, owner GtkNotebook)

(tbstudio:7): Gtk-CRITICAL **: 14:26:02.267: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkNotebook

(tbstudio:7): Gtk-CRITICAL **: 14:26:02.269: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkScrollbar

(tbstudio:7): Gtk-CRITICAL **: 14:26:02.269: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkScrollbar

(tbstudio:7): Gtk-WARNING **: 14:26:02.270: Negative content height -1 (allocation 1, extents 1x1) while allocating gadget (node notebook, owner GtkNotebook)

(tbstudio:7): Gtk-CRITICAL **: 14:26:02.270: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkNotebook

(tbstudio:7): Gdk-ERROR **: 14:26:02.422: The program 'tbstudio' received an X Window System error.
This probably reflects a bug in the program.
The error was 'GLXBadContext'.
  (Details: serial 9352 error_code 158 request_code 152 (GLX) minor_code 6)
  (Note to programmers: normally, X errors are reported asynchronously;
   that is, you will receive the error a while after causing it.
   To debug your program, run it with the GDK_SYNCHRONIZE environment
   variable to change this behavior. You can then get a meaningful
   backtrace from your debugger if you break on the gdk_x_error() function.)
Trace/breakpoint trap (core dumped)

Thanks for help
